build.go

```diff
@@ -255,6 +257,26 @@ func (cmd *buildCommand) Run(args []string) (err error) {
 	eg.Go(func() error {
 		return sess.Run(ctx, sessDialer)
 	})
+
+	var cacheExports []*controlapi.CacheOptionsEntry;
+	cacheExports = append(cacheExports, &controlapi.CacheOptionsEntry{
+		Type:  "inline",
```

Comment:
It seems non-ideal to set it to always inline, my logic is that this should follow as closely as possible the behavior of `docker build`. I suppose another option would be to default it to on, but let it be configurable - or even to expose the full buildctl style export cache arguments to allow exporting to separate images.

I'm not really sure what the overhead of the cache in the image is. If it's small enough I'd be inclined to leave it.

Reply:
I was looking forward using `mode=max` for `--export-cache` since it exports multi-stage build layers.